packages/pglite/src/parse.ts

```diff
@@ -13,50 +13,61 @@ import { parseType } from "./types.js";
  */
 export function parseResults(
   messages: Array<BackendMessage>,
-  options?: QueryOptions
+  options?: QueryOptions,
 ): Array<Results> {
   const resultSets: Results[] = [];
-  let currentResultSet: Results | null = null;
+  let currentResultSet: Results = { rows: [], fields: [] };
+  let affectedRows = 0;
 
-  for (const msg of messages) {
+  const filteredMessages = messages.filter(
```

Comment:
filtering because there are messages after the last `CommandCompletedMessage` that we don't care about, but we need to check to see if, when we're iterating, if we're on the last `CommandCompletedMessage`, so that we can append `affectedRows`.